packages/ui/src/kit/VInputGroup/VInputGroup.vue

```diff
@@ -10,11 +10,22 @@ import ThemeMixin from '@/mixins/ThemeMixin';
 export default {
   name: 'VInputGroup',
 
+  props: {
+    skin: {
+      type: String,
+      default: 'primary',
+      validator(value) {
+        return ['primary', 'secondary'].indexOf(value) !== -1;
+      },
+    },
+  },
+
   computed: {
     vInputGroupCssClass() {
       return {
         ...this.themeCssClass,
         'v-input-group': true,
+        [this.skin]: true,
```

Comment:
Why in `input-group` you use `{skin}`, but in `select-atom` using construction like `skin-{this.skin}`? Maybe change it to the same style, in `select-atom` (seems to we prefer just skin-name instead of `skin` prefix)?